AlphaWallet/Tokens/ViewControllers/NewTokenViewController.swift

```diff
@@ -314,20 +325,28 @@ class NewTokenViewController: UIViewController {
         let symbol = symbolTextField.value
         let decimals = Int(decimalsTextField.value) ?? 0
         guard let tokenType = self.tokenType else { return }
-        //TODO looks wrong to mention ERC875TokenBalance specifically
-        var balance: [String] = viewModel.erc875TokenBalance
+
+        var nonFungibleBalance: NonFungibleBalance
+        switch tokenType {
+        case .erc875:
+            if let balance = viewModel.nonFungibleBalance {
+                nonFungibleBalance = balance.rawValue.isEmpty ? .erc875(["0"]) : balance
```

Comment:
Should this be `.erc875([])` or `.erc875([0])`? Ditto for 2 lines below this.

Reply:
actually i don't know, previous implementation setting initial value as "0", if array is empty